CRM/Upgrade/Incremental/sql/5.56.alpha1.mysql.tpl

```diff
@@ -1,5 +1,23 @@
 {* file to handle db changes in 5.56.alpha1 during upgrade *}
 
+-- Add in Year prior to previous fiscal year and Previous 2 fiscal years
+SELECT @option_group_id_date_filter := max(id) from civicrm_option_group where name = 'relative_date_filters';
+UPDATE `civicrm_option_value` SET weight = 64 WHERE option_group_id = @option_group_id_date_filter AND weight = 62;
+UPDATE `civicrm_option_value` SET weight = 63 WHERE option_group_id = @option_group_id_date_filter AND weight = 61;
+UPDATE `civicrm_option_value` SET weight = 62 WHERE option_group_id = @option_group_id_date_filter AND weight = 60;
+UPDATE `civicrm_option_value` SET weight = 61 WHERE option_group_id = @option_group_id_date_filter AND weight = 59;
+UPDATE `civicrm_option_value` SET weight = 59 WHERE option_group_id = @option_group_id_date_filter AND weight = 58;
+UPDATE `civicrm_option_value` SET weight = 58 WHERE option_group_id = @option_group_id_date_filter AND weight = 57;
+UPDATE `civicrm_option_value` SET weight = 57 WHERE option_group_id = @option_group_id_date_filter AND weight = 56;
+UPDATE `civicrm_option_value` SET weight = 56 WHERE option_group_id = @option_group_id_date_filter AND weight = 55;
+UPDATE `civicrm_option_value` SET weight = 55 WHERE option_group_id = @option_group_id_date_filter AND weight = 54;
+INSERT INTO
```

Comment:
`INSERT IGNORE INTO`
in case somebody reruns the upgrade, or has already made this customization which believe it or not is supported/documented: https://docs.civicrm.org/user/en/latest/searching/relative-date-formats/#adding-new-filters

Reply:
Done, thanks.